conf/application.conf

```diff
@@ -204,6 +204,7 @@ go-live-date = "2022-03-01"
 features {
   isPreLaunch = false
   ukCompanyPrivateBeta = true
+  partnershipEnabled = true
```

Comment:
is this set false somewhere else, as I think these are the defaults

Reply:
Have left this as true as default as I believe we will only want to set this to false in a specific environment?

Reply:
We probably want to make sure it's disabled in prod. @ajchilts are you ok to create another PR for this?

Reply:
Sure 👍

Reply:
@RyanKellyHMRC  - PR for production config: https://github.com/hmrc/app-config-production/pull/14040